### ISSUE TITLE
Shell extension conflict

### ISSUE THREAD

**Issue by Ojlkalfapf** [crystalidea/anytoiso#35]
Enabling shell extension causes explorer crash when right-click on a drive in left pane of Explorer\My Computer window.
This problem is caused by a conflict with nvidia shell extension.

**Comment by kleuter** [comment]
True, we added a recent workaround in our shell extension not to crash with NVIDIA shell extension (in our dev build), but it still needs to be disabled in order for AnyToISO to work properly
